### ISSUE TITLE
loss function doubt

### ISSUE THREAD

**Issue by gissemari** [phreeza/tensorflow-vrnn#4]
Is _dec_rho_ replaced by the value of _dec_sigma_ when calculating _lossfunc = get_lossfunc(...)_ for a specific reason or it is just a bug?

**Comment by phreeza** [comment]
Definitely looks like a bug, thanks for catching it. Should be fixed now.
